client/src/components/Pcards.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Card from 'react-bootstrap/Card'
+import CardColumns from 'react-bootstrap/CardColumns'
+
+const Pcards = ({ projects }) => {
+    return (
+        <div>
+            <CardColumns>
+                {projects.map((projects) => (
+                    <Card>
+                        <Card.Img variant="top" src={"http://localhost:8000" + projects.images[0].file_path + projects.images[0].file_name + projects.images[0].file_type} />
```

Comment:
I believe this is going to need to change to accommodate the file system on aws. I'll look into that on a separate branch.